client/app/components/mails_input.coffee

```diff
@@ -127,11 +128,12 @@ module.exports = MailsInput = React.createClass
             else
                 event.dataTransfer.dropEffect = 'none'
 
+        # Show field if results
+        className += ' shown' if @state.focus
```

Comment:
Beware, focus and shown are two different things, this could cause confusion in the future.